compute/admin_guide/tools/twistcli.adoc

```diff
@@ -113,106 +113,186 @@ Collects log data from a node and uploads it to Prisma Cloud's support area.
 The _twistcli_ tool offers feature parity across all supported operating systems, with a few exceptions.
 The following table highlights where functions are disabled, or work differently, on a given platform.
 
-[cols=".^2,2,1,1,1", frame="topbot"]
+[cols=".^2,2,1,1,1,1,1", frame="topbot"]
 |====
-2+^| twistcli 3+^| Platform
+2+^| twistcli 5+^| Platform
 
 |Command
 |Subcommand
 |Linux
+|Linux ARM64
 |macOS
+|macOS ARM64
 |Windows
 
 .3+|`console` {set:cellbgcolor:#fff}
 |`export`
 |Yes
+|No
+|Yes
 |Yes
 |Yes
 
 |`install`
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No# {set:cellbgcolor:#000}
+|No
+|No
+|No
+|No
 
 |`uninstall` {set:cellbgcolor:#fff}
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No# {set:cellbgcolor:#000}
+|No
+|No
+|No
+|No
 
 .3+|`defender` {set:cellbgcolor:#fff}
 |`export`
 |Yes
+|Yes: Host AWS; Daemonsets (AWS and GCP)
+|Yes
 |Yes
 |Yes
 
 |`install`
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No#
+|Yes
+|No
+|No
+|No
 
 |`uninstall` {set:cellbgcolor:#fff}
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No#
+|No
+|No
+|No
+|No
 
 |`hosts` {set:cellbgcolor:#fff}
 |`scan`
 |Yes
-|[white]#No^1^# {set:cellbgcolor:#000}
-|[white]#No#
+|Yes
+|No^1^
+|No
+|No
 
 |`images` {set:cellbgcolor:#fff}
 |`scan`
 |Yes
+|No
 |Yes^2^
+|Yes
 |Yes^3^
 
 .2+|`intelligence`
 |`upload`
 |Yes
+|Not tested
 |Yes
+|Not tested
 |Yes
 
 |`download`
 |Yes
+|Not tested
+|Yes
 |Yes
 |Yes
 
 |`pcf`
 |`scan`
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No# {set:cellbgcolor:#000}
+|No
+|No
+|No
+|No
 
 |`app-embedded` {set:cellbgcolor:#fff}
 |`embed`
 |Yes
+|No
 |Yes
+|No
 |Yes
 
 |`restore` {set:cellbgcolor:#fff}
 |
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No# 
+|Not tested
+|No
+|No
+|No
 
 |`serverless` {set:cellbgcolor:#fff}
 |`scan`
 |Yes
+|No
 |Yes
+|No
 |Yes
 
 .2+|`support`
 |`dump`
 |Yes
-|[white]#No^4^# {set:cellbgcolor:#000}
-|[white]#No^4^#
+|Not tested
+|No^4^
+|No
+|No^4^
 
 |`upload` {set:cellbgcolor:#fff}
 |Yes
+|Not tested
+|Yes
+|Yes
+|Yes
+
+|`tas`
+|`scan`
+|Yes
+|Yes
+|Yes
 |Yes
 |Yes
 
+|`coderepo` {set:cellbgcolor:#fff}
+|`scan`
+|Yes
+|Yes
+|Yes
+|Yes
+|Yes
+
+.3+|`clustered-db`
```

Comment:
@ssugandh remove the clustered-db section, we are exposing this can capability as little as possible (it was dedicated to a very specific customer), and it was also not tested on all platforms